torchvision/ops/_masks_to_bounding_boxes.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+
+def masks_to_bounding_boxes(masks: torch.Tensor) -> torch.Tensor:
```

Comment:
Since all other utils use the term `box` instead of `bounding_box`, let's call this `masks_to_boxes` instead
(although I would personally prefer the way you did it :)  )